package.json

```diff
@@ -6,12 +6,16 @@
   "author": "Nikhil Arora <narora200@gmail.com>, Saumya Tiwari <saumyatiwari.29@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "start:dev": "echo \"starting the-remotants backend\" && nodemon src",
+    "start": "babel-node -- src",
```

Comment:
`babel-node` transpiles the files on the fly. Make sure to use it only in development.